fw/http_parser.c

```diff
@@ -11235,6 +11295,21 @@ do {										\
 		__FSM_I_MOVE_fixup(Resp_I_EoT, __fsm_n, 0);
 	}
 
+	__FSM_REQUIRE_FIRST_DIGIT(Resp_I_CC_StaleIfErrorVBeg,
+				  Resp_I_CC_StaleIfErrorV);
+
+	__FSM_STATE(Resp_I_CC_StaleIfErrorV) {
+		__fsm_sz = __data_remain(p);
```

Comment:
Should we don like for previos directives?

if (unlikely(resp->cache_ctl.flags & TFW_HTTP_CC_STALE_IF_ERROR))
	__FSM_EXIT(T_DROP);

Reply:
Why? We can have repeating headers with different values

Reply:
Are you sure? I check that without this check we can set cache_control: stale-if-error=20, stale-if-error=25. Is it ok?
For other cache-control directives it is prohibited.